src/shogun/base/init.cpp

```diff
@@ -147,6 +154,26 @@ namespace shogun
 		return sg_io;
 	}
 
+	float64_t get_global_fequals_epsilon()
+	{
+		return sg_fequals_epsilon;
```

Comment:
@vigsterkr this is fucked up, but the way it was before was even more so. We should just get rid of all of this accuracy business and rather fix the serialization, which should not be lossy (in my eyes)